graphbolt/include/csc_sampling_graph.h

```diff
@@ -0,0 +1,140 @@
+/**
+ *  Copyright (c) 2023 by Contributors
+ * @file graphbolt/include/csc_sampling_graph.h
+ * @brief Header file of csc sampling graph.
+ */
+
+#include <torch/custom_class.h>
+#include <torch/torch.h>
+
+#include <string>
+#include <vector>
+
+namespace graphbolt {
+namespace sampling {
+
+using StringList = std::vector<std::string>;
+
+/**
+ * @brief Heterogeneous information in the Graph.
```

Comment:
Need more docstring. Please describe (e.g., using an example) what's the concrete semantics of each member.

Reply:
added